src/dragonball/src/error.rs

```diff
@@ -14,6 +14,38 @@ use dbs_virtio_devices::Error as VirtIoError;
 
 use crate::device_manager;
 
+/// Shorthand result type for internal VMM commands.
+pub type Result<T> = std::result::Result<T, Error>;
+
+/// Errors associated with the VMM internal logic.
+///
+/// These errors cannot be generated by direct user input, but can result from bad configuration
+/// of the host (for example if Dragonball doesn't have permissions to open the KVM fd).
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    /// Cannot open /dev/kvm. Either the host does not have KVM or Dragonball does not have
+    /// permission to open the file descriptor.
```

Comment:
Sorry, this comment is not compatible with the error.
The error `Kvm(#[source] kvm_ioctls::Error)` here refers to errors occurs during kvm-ioctls instead of /dev/kvm file oprations related error.

Reply:
I have updated the comment